### PR TITLE
feat: Allow to see the total feedback of a teacher with given teacher name and department name

### PR DESCRIPTION
**Pull request by VirtualRcoder**
some files also modified and removed

### CODE REVIEW

File: Feedback/templates/Feedback/teacher_check.html [hunk 1]
```diff
@@ -4,45 +4,63 @@
 {%load static%}
 
 {% block feedback %}
-    <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <h1 align="center"class="check-h1">Feedback Details</h1><br><br>
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here please give space between `align` property and `class` property

File: Feedback/templates/Feedback/teacher_check.html [hunk 2]
```diff
@@ -4,45 +4,63 @@
 {%load static%}
 
 {% block feedback %}
-    <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <h1 align="center"class="check-h1">Feedback Details</h1><br><br>
+  <form action="{%url 'teacher_check'%}" method="POST" class="check-form">
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: Please give space after `{%` and before `%}` in action property.

File: Feedback/templates/Feedback/teacher_check.html [hunk 3]
```diff
@@ -4,45 +4,63 @@
...
-    <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <h1 align="center"class="check-h1">Feedback Details</h1><br><br>
+  <form action="{%url 'teacher_check'%}" method="POST" class="check-form">
+    {%csrf_token%}
+	  <div class="form">		
+	    <h3>
+		  Department:-
+			<input list="teacher_dept" name ="dropdown" autocomplete="off">
+			  <datalist id="teacher_dept"class="check-datalist">
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here give space between `id` property and `class` property.

File: Feedback/templates/Feedback/teacher_check.html [hunk 4]
```diff
@@ -4,45 +4,63 @@
...
 {% block feedback %}
-    <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <h1 align="center"class="check-h1">Feedback Details</h1><br><br>
+  <form action="{%url 'teacher_check'%}" method="POST" class="check-form">
+    {%csrf_token%}
+	  <div class="form">		
+	    <h3>
+		  Department:-
+			<input list="teacher_dept" name ="dropdown" autocomplete="off">
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: remove space after `name`

File: Feedback/templates/Feedback/teacher_check.html [hunk 5]
```diff
@@ -4,45 +4,63 @@
...
+			  <datalist id="teacher_dept"class="check-datalist">
+				<option selected="disabled" value="none"></option>
+				  {% for element in teacher %}
+					<option value="{{ element.teacher_dept }}"></option>
+				  {%endfor%}
+			  </datalist>
+			</input>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+		  Faculty:-
+			<input list="teacher_name" name='dropdown1' autocomplete="off">
+			  <datalist id="teacher_name"class="check-datalist">
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here give space between `id` and `class` property

File: Feedback/templates/Feedback/teacher_check.html [hunk 6]
```diff
@@ -4,45 +4,63 @@
...
+				<option selected="disabled" value="none"></option>
+				  {% for element in teacher %}
+					<option value="{{ element.teacher_name }}"></option>
+				  {%endfor%}
+			  </datalist>
+			</input>
+		</h3>
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: shouldn't this tag be given after two spaces

File: Feedback/templates/Feedback/teacher_check.html [hunk 7]
```diff
@@ -4,45 +4,63 @@
...
+		</div><br><br>	
+		<button class="btn btn-primary check-button"name="submit" type="submit" name = 'Submit'  >Submit</button><br><br>
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here give space between `class` and `name` property.
Why have you given two `name` values?
And remove space before `>` in opening button tag.

File: Feedback/templates/Feedback/teacher_check.html [hunk 8]
```diff
@@ -4,45 +4,63 @@
...
+		
+		{%if Flag == 1%}
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here give space after `{%` and before `%}`

File: Feedback/templates/Feedback/teacher_check.html [hunk 9]
```diff
@@ -4,45 +4,63 @@
...
+			<div class="detail">
+				<center><h1 class="check-detail-center-h1">Details of {{faculty}}</h1><p class="check-detail-center-p">({{department}})</p></center>
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here give space after `{{` and before `}}`

File: Feedback/templates/Feedback/teacher_check.html [hunk 10]
```diff
@@ -4,45 +4,63 @@
...
-				{%for F in Feedback%}
-					{%if F.teacher.teacher_name == faculty and F.teacher.teacher_dept == department%}
-					<tr><td >{{F.section}}</td><td>{{F.Punctuality}}</td><td>{{F.Subject_knowledge}}</td><td>{{F.Behaviour}}</td><td>{{F.Method_of_teaching}}</td><td>{{F.Audibility}}</td><td>{{F.Syllabus_coverage}}</td></tr>
-					{%endif%}
-				{%endfor%}
-			</table>
-	</div>			
+				<table>
+					 <thead>
+						<th >Section</th>
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: Why have you given this much space for identation here? It should be consistent throughout the file.
if you are using two spaces for identation then use two only for every tag.

File: Feedback/templates/Feedback/teacher_check.html [hunk 11]
```diff
@@ -4,45 +4,63 @@
...
+						<th>Punctuality</th>
+						<th>Subject_knowledge</th>
+						<th>Behaviour</th>
+						<th>Method_of_teaching</th>
+						<th>Audibility</th>
+						<th>Syllabus_coverage</th>
+					</thead>
+					{%for F in Feedback%}
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here give space after {% and before %}

File: Feedback/templates/Feedback/teacher_check.html [hunk 12]
```diff
@@ -4,45 +4,63 @@
...
+						{%if F.teacher.teacher_name == faculty and F.teacher.teacher_dept == department%}
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: Same here give spaces after {% and before %}

File: Feedback/templates/Feedback/teacher_check.html [hunk 13]
```diff
@@ -4,45 +4,63 @@
...
+								<td >{{F.section}}</td>
+								<td>{{F.Punctuality}}</td>
+								<td>{{F.Subject_knowledge}}</td>
+								<td>{{F.Behaviour}}</td>
+								<td>{{F.Method_of_teaching}}</td>
+								<td>{{F.Audibility}}</td>
+								<td>{{F.Syllabus_coverage}}</td>
+							</tr>
+						{%endif%}
+					{%endfor%}
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here also

File: Feedback/templates/Feedback/teacher_check.html [hunk 14]
```diff
@@ -4,45 +4,63 @@
...
+				</table>
+			</div>			
 		{%elif Flag == 0%}
 			<script>window.alert("Teacher feedback does not exist");</script>
 		{%endif%}
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here also in both `elif` and `endif` tag

File: Feedback/static/Feedback/base.css [hunk 15]
```diff
@@ -1,124 +1,131 @@
...
+.feed-button{
+  font-size: 20px;margin-top: 30px;
 }
 
-thead{
-    line-height: 30px;
-    background: #2e63e7;
-    text-transform: uppercase;
+.feed-h1{
+  margin-top: 50px; margin-bottom: 50px;color:white;text-decoration: underline; text-shadow: 2px 2px #05a151;font-weight: bold;letter-spacing: 2px;
```
Comment (gutsytechster) [inline Feedback/static/Feedback/base.css]: Please write CSS properties in different lines here.

File: Feedback/static/Feedback/base.css [hunk 16]
```diff
@@ -1,124 +1,131 @@
...
-    vertical-align:middle;
-    font-weight:300;
-    font-size:15px;
-    text-shadow: -1px -1px 1px rgba(0, 0, 0, 0.1);
-    border-right: 1px solid #C1C3D1;
-    line-height: 18px;  
-}
\ No newline at end of file
+.check-detail-center-h1{
+  font-weight: bold;text-decoration:underline;
```
Comment (gutsytechster) [inline Feedback/static/Feedback/base.css]: here too

File: Feedback/static/Feedback/base.css [hunk 17]
```diff
@@ -1,124 +1,131 @@
...
+.check-button{
+  margin-top: 20px;margin-left: 775px;  width: 5%;font-size: 15px;
+}
+
+.check-form{
+  margin-top: 30px;margin-left: 160px;
+}
+
+.check-h1{
+  margin-top: 50px;color:white;text-decoration: underline; text-shadow: 2px 2px #05a151;font-weight: bold;letter-spacing: 2px;
```
Comment (gutsytechster) [inline Feedback/static/Feedback/base.css]: And in all of the above

**Comment by VirtualRcoder** [comment]
Okay

File: Feedback/static/Feedback/base.css (hunk 17 above)
Comment (gutsytechster) [inline Feedback/static/Feedback/base.css]: And you have not given spaces after colon(:) in many fields. So please do give

File: Feedback/static/Feedback/teacher_check.css [hunk 18]
```diff
@@ -1,13 +1,13 @@
...
-    transform: scale(1.2); 
-    box-shadow: 0px 3px 7px rgba(0, 0, 0, 0.5);
+  color: #000;
+  background: rgba(255, 255, 255, 0.7);
+  font-weight: 900;
+  border-top: 1px solid #22262e;
+  transition: .4s ease-in; 
+  transform: scale(1.2); 
+  box-shadow: 0px 3px 7px rgba(0, 0, 0, 0.5);
 }
```
Comment (gutsytechster) [inline Feedback/static/Feedback/teacher_check.css]: here add a new line at the end of the file.

File: Feedback/templates/Feedback/teacher_check.html (hunk 10 above)
Comment (VirtualRcoder) [reply Feedback/templates/Feedback/teacher_check.html]: It is showing consistent through all file in my laptop.

File: Feedback/templates/Feedback/teacher_check.html [hunk 19]
```diff
@@ -0,0 +1,66 @@
...
+	<table>
+	  <thead>
+	    <th>Section</th>
+	    <th>Punctuality</th>
+	    <th>Subject_knowledge</th>
+	    <th>Behaviour</th>
+	    <th>Method_of_teaching</th>
+	    <th>Audibility</th>
+	    <th>Syllabus_coverage</th>
+          </thead>
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: The `<thead>` tag is not idented properly here

File: Feedback/templates/Feedback/teacher_check.html [hunk 20]
```diff
@@ -0,0 +1,66 @@
...
+	    <datalist id="teacher_name" class="check-datalist">
+              <option selected="disabled" value="none"></option>
+	        {% for element in teacher %}
+	          <option value="{{ element.teacher_name }}"></option>
+		{% endfor %}
+	    </datalist>
+	  </input>
+      </h3>
+    </div><br><br>	
+    <button class="btn btn-primary check-button" name="submit" type="submit" name='Submit'>Submit</button><br><br>
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: Why have you used two `name` values here?

File: Feedback/templates/Feedback/teacher_check.html [hunk 21]
```diff
@@ -0,0 +1,66 @@
...
+    <div class="form">		
+      <h3>
+        Department:-
+	  <input list="teacher_dept" name="dropdown" autocomplete="off">
+	    <datalist id="teacher_dept" class="check-datalist">
+	      <option selected="disabled" value="none"></option>
+		{% for element in teacher %}
+		  <option value="{{ element.teacher_dept }}"></option>
+		{% endfor %}
+            </datalist>
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: here `</datalist>` tag is not matching the opening tag

File: Feedback/templates/Feedback/teacher_check.html [hunk 22]
```diff
@@ -0,0 +1,66 @@
...
+
+{% block feedback %}
+  <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <h1 align="center" class="check-h1">Feedback Details</h1><br><br>
+  <form action="{% url 'teacher_check' %}" method="POST" class="check-form">
+    {% csrf_token %}
+    <div class="form">		
+      <h3>
+        Department:-
+	  <input list="teacher_dept" name="dropdown" autocomplete="off">
```
Comment (gutsytechster) [inline Feedback/templates/Feedback/teacher_check.html]: The `<input` tag here should be indented after two spaces

**Pull request merged**